lib/puppetserver/ca/action/generate.rb

```diff
@@ -143,15 +140,25 @@ def generate_pki(settings, signing_digest, subject_alt_names = '')
             [settings[:cakey], int_key],
           ]
 
+          errors = FileSystem.check_for_existing_files(public_files.map { |f| f.first })
+          errors += FileSystem.check_for_existing_files(private_files.map { |f| f.first })
+
+          if errors.any?
```

Comment:
Definitely more readable but I'm still don't love using `any?` like this

Reply:
I missed the conversation on why that is last time. What don't you like about it? (I'm happy to change it, just would like to know what the misgivings are.)

Reply:
Because it's only the same as `!empty?` if the collection can contain _only_ truthy values).

When it's unambiguous that the collection can only contain truthy values (like here) then I'm fine with it.

My not-love for it comes from a concern that using it too freely will encourage us in the future to use it as a synonym for not-empty even in collections that can contain falsey values.

Reply:
^ @justinstoller why don't you like any here?